### PR TITLE
Create ui-c8y-1020-26-0-add-new-branding-editor-7017.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by tba

Ticket: [MTM-53766](https://cumulocity.atlassian.net/browse/MTM-53766)
Original PR: [7017](https://github.softwareag.com/IOTA/cumulocity-ui/pull/7017)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: add new branding editor (#7017)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: New branding editor for customizing the look and feel
```

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-53766
+version: 1020.26.0
+---
+add new branding editor (#7017)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:17]: The Claude AI suggested to use the following body:
```suggestion
To enable users to easily customize the look and feel of their applications, a new branding editor has been added. The branding editor provides a user-friendly interface for modifying colors, fonts, logos and other visual elements without requiring any coding skills. This change allows users to create a unique and consistent branding across their applications, improving the overall user experience and making it easier to align the application's appearance with corporate design guidelines.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md (hunk 2 above)
Comment (reey) [reply content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:17]:
```suggestion
The branding editor allows users to create a unique and consistent branding across their applications, improving the overall user experience and making it easier to align the application's appearance with corporate design guidelines.

The branding editor has been enhanced. It now supports:
* maintaining multiple branding variants
* easy switching between branding variants
* brand apps differently
* adding custom CSS
* editing the branding JSON object directly from the UI
* import/export of branding variants
* invalidating cookie consents when a new privacy policy is provided
* dark theme
```

**Review by reey — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-53766
+version: 1020.26.0
+---
+The branding editor allows users to create a unique and consistent branding across their applications, improving the overall user experience and making it easier to align the application's appearance with corporate design guidelines.
+
+The branding editor has been enhanced. It now supports:
+* maintaining multiple branding variants
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:20]:
```suggestion
* Maintaining multiple branding variants
```

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+* easy switching between branding variants
+* branding apps differently
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:22]:
```suggestion
* Branding apps differently
```

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+* adding custom CSS
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:23]:
```suggestion
* Adding custom CSS
```

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+* editing the branding JSON object directly from the UI
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:24]:
```suggestion
* Editing the branding JSON object directly from the UI
```

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+* importing/exporting branding variants
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:25]:
```suggestion
*Importing/exporting branding variants
```

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+* invalidating cookie consents when a new privacy policy is provided
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:26]:
```suggestion
* Invalidating cookie consents when a new privacy policy is provided
```

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+* dark theme
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md:27]:
```suggestion
* Dark theme
```

**Comment by BeateRixen** [comment]
@rahultsag Please review and approve

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md (hunk 7 above)
Comment (rahultsag) [reply content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md]: Seems space is missing 

c8y-docs 2024-09-10 10-28-33

File: content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+The branding editor has been enhanced. It now supports:
+* Maintaining multiple branding variants
+* Easy switching between branding variants
+* Branding apps differently
+* Adding custom CSS
+* Editing the branding JSON object directly from the UI
+*Importing/exporting branding variants
+* Invalidating cookie consents when a new privacy policy is provided
+* Dark theme
```
Comment (rahultsag) [inline content/change-logs/application-enablement/ui-c8y-1020-26-0-add-new-branding-editor-7017.md]:
```suggestion
We have made significant improvements to our Branding Editor, offering a more robust and user-friendly experience for managing brand identity across multiple applications.

**New functionalities:**
- Configure unique branding for individual applications
- Create, maintain, and easily switch between different branding configurations
- Edit branding JSON objects directly from the UI and add custom CSS
- Import/export branding variants
- Automatically invalidate cookie consents when updating privacy policies
- Implement light and dark theme choices
- Generate color shades based on primary brand color with fine-tuning options

**Functional improvements:**
- More intuitive interface for managing brand identity
- Enhanced flexibility in customizing and applying branding elements
```

@reey proposing this changelog for the new branding manager

**Review by BeateRixen — APPROVED** [review]

**Review by rahultsag — APPROVED** [review]

**Pull request merged**